cmd/notary/tuf.go

```diff
@@ -257,9 +258,10 @@ func verify(cmd *cobra.Command, args []string) {
 	}
 
 	// Create hasher and hash data
-	stdinHash := fmt.Sprintf("sha256:%x", sha256.Sum256(payload))
-	serverHash := fmt.Sprintf("sha256:%s", target.Hashes["sha256"])
-	if stdinHash != serverHash {
+	stdinHash := sha256.Sum256(payload)
+	serverHash := target.Hashes["sha256"]
```

Comment:
Does this need a check to verify that there's a sha256 entry? The Sprintf would have been resilient to this case.